test/typescript-tests/testTypes.ts

```diff
@@ -1282,6 +1282,17 @@ Matrices examples
     assert.strictEqual(math.matrix([1, 2, 3]) instanceof math.Matrix, true)
   }
 
+  // Eigenvalues and eigenvectors (should this be extended?)
```

Comment:
👍 I think this covers the case, can you remove the question "should this be extended?"

Reply:
Done.

Reply:
Thanks!